### PR TITLE
Add recipe "Roulades with Pretzel Dumplings"

### PR DESCRIPTION
**Pull request by aosthof**
Didn't imagine it'd take me hours to come up with a first version, I only realized it's hard when translating stuff from my head into hopefully proper English ;)

### CODE REVIEW

File: recipes/entrees/roulade_pretzel_dumplings.adoc [hunk 1]
```diff
@@ -0,0 +1,144 @@
+[id='sec.roulades_pretzel_dumplings']
+
+ifdef::env-github[]
+:imagesdir: ../images
```
Comment (jeremy-moffitt) [inline recipes/entrees/roulade_pretzel_dumplings.adoc]: this needs ../../images
blame the refactor, sorry!

**Review by jeremy-moffitt — CHANGES_REQUESTED** [review]
Looks amazing built locally, needs a slight tweak to render the pictures correctly on github (see https://github.com/SUSE/recipebook/blob/063c40bc51cfa0f417de8321e05cf080fecd8f02/recipes/entrees/roulade_pretzel_dumplings.adoc  )

File: recipes/entrees/roulade_pretzel_dumplings.adoc (hunk 1 above)
Comment (aosthof) [reply recipes/entrees/roulade_pretzel_dumplings.adoc]: Fixed now.

**Review by rsimai — APPROVED** [review]
looks lecker to me

**Review by rsimai — APPROVED** [review]
renders nicely, looks great!

**Review by jeremy-moffitt — APPROVED** [review]

**Pull request merged**